github/ci/prow-deploy/tasks/deploy.yml

```diff
@@ -28,6 +28,13 @@
     KUBECONFIG: "{{ kubeconfig_path }}"
   when: bootstrap_full_config
 
+- name: Delete priority classes on prow-workloads BM cluster
+  when: delete_priority_classes
+  shell: |
+    kubectl delete pc sriov vgpu
```

Comment:
once we have the new sriov-place-holder pc we would need to add it here as well

Reply:
btw you might want to delete -f the pc file
it will auto delete all and just what is there

Reply:
> once we have the new sriov-place-holder pc we would need to add it here as well

Or add `--ignore-not-found=true`

Reply:
> btw you might want to delete -f the pc file it will auto delete all and just what is there (auto updating, but not backward compatible, in case we change the file by removing a pc)

@enp0s3 keep me honest, but we discussed that using the file is not possible, as it's already the new configuration, thus it will fail to remove older stuff (i.e. if something is removed)